engine/components-api/src/main/java/com/cloud/template/TemplateManager.java

```diff
@@ -48,6 +48,8 @@
     static final ConfigKey<Integer> TemplatePreloaderPoolSize = new ConfigKey<Integer>("Advanced", Integer.class, TemplatePreloaderPoolSizeCK, "8",
             "Size of the TemplateManager threadpool", false, ConfigKey.Scope.Global);
 
+
```

Comment:
white spaces, no relevant changes in this file. can remove all the changes in this file?